app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I don't think we're using this.

Reply:
Do we need to remove from gov template as well as from register?